Coding/fw/lib/Lawicel/NVMInterface.h

```diff
@@ -33,6 +33,17 @@ class NVMInterface
 public:
     /* CONSTANTS ******************************************************************************/
 
+    /**
+    * Key Definition (Addresses) to be read by NVM Adapter for initialization
+    */
+    const String INIT_SERIAL_BAUD = "SerialBaud"; /**< Initial Serial Baudrate*/
```

Comment:
these should also be static and inititalized in cpp file. Otherwise any instance of the class will create its own copy of strings in construction

Reply:
It is part of the interface. There is no cpp file. Where should these be then? Also, there is also one instance, which is the NVM Adapter